File: app/src/main/java/com/checkpoint/qr/detector/events/OnFinishDetectionEvent.java
package com.checkpoint.qr.detector.events;

public class OnFinishDetectionEvent extends BaseEvent {

    public String direction;
    public String translation;

    public OnFinishDetectionEvent(String direction,String translation) {
        this.direction = direction;
        this.translation = translation;
    }

    public String getResultDirection() {

        return direction;

    }

    public String getResultTranslation() {
        return translation;
    }
}

File: app/src/main/java/com/checkpoint/qrdetector/events/OnFinishDetectionEvent.kt
package com.checkpoint.qrdetector.events

import android.graphics.Bitmap

class OnFinishDetectionEvent(private val direction: String,
                             private val translate: String?,
                             private val image: Bitmap
) : EventBase() {


    fun getResultDirection(): String? {
        return direction
    }

    fun getResultTranslation(): String? {
        return translate
    }
    fun getImageBase64():Bitmap?{
        return image
    }

}